src/views/primary-auth/PrimaryAuthForm.js

```diff
@@ -153,7 +174,27 @@ export default Form.extend({
       autoComplete: 'current-password',
       // TODO: support a11y attrs in Courage - OKTA-279025
       render: function() {
-        this.$(`input[name=${this.options.name}]`).prop('required', true);
+        const that = this;
```

Comment:
@pradeepdewda-okta Having some trouble with refactoring (haven't worked with jQuery in a while and Backbone/Courage ever). I'm going to add a TODO for now because I need shift focus to the other a11y fix for this week's release.